sportsreference/ncaaf/teams.py

```diff
@@ -419,13 +610,30 @@ def first_downs(self):
         """
         return self._first_downs
 
+    @float_property_decorator
+    def opponents_first_downs(self):
+        """
+        Returns a ``float`` of the opponents' total number of first downs
+        achieved per game.
+        """
+        return self._opponents_first_downs
+
     @float_property_decorator
     def penalties(self):
         """
-        Returns a ``float`` the average number of penalties conceded per game.
+        Returns a ``float`` of the average number of penalties conceded per
+        game.
```

Comment:
Similar to above - was this meant to be wrapped on a new line?

Reply:
The main change here was just the small grammar correction of adding `of` - I think I had to wrap the line so as not to exceed your line width of 79 chars

Reply:
Ah yes, I completely missed that! Thanks for the change!